CocoaUPnP/Networking/SSDP/SSDPServiceBrowser.m

```diff
@@ -22,9 +22,11 @@
 //  Expanded for CocoaUPnP by A&R Cambridge Ltd, http://www.arcam.co.uk
 //  Copyright 2015 Arcam. See LICENSE file.
 
+@import UIKit;
```

Comment:
Ahh. Ok. Haven't been looking at the OSX side of things. 

BTW, love Arcam gear. Currently using a FMJ400. :-)